src/main/java/me/moodcat/database/entities/Room.java

```diff
@@ -45,28 +51,53 @@
     private Integer id;
 
     /**
+     * The name of the room.
+     */
+    @Column(name = "roomName")
+    private String roomName;
```

Comment:
Frontend uses the field name (later declared in this class) https://github.com/MoodCat/MoodCat.me-Frontend/blob/master/app/views/room.html#L3